app/views/layouts/application.html.erb

```diff
@@ -22,19 +22,22 @@
 
     <%# Includes all stylesheet files in app/assets/stylesheets %>
     <%= stylesheet_link_tag :app, 'data-turbo-track': 'reload' %>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/sul-dlss/component-library@v2025-01-10/styles/sul.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/sul-dlss/component-library@main/styles/sul.css">
```

Comment:
Do you think we should keep this pointing at `main` or can we use the new `v2025-02-03` tag?

Reply:
I just released v2025-02-03 with the intention of doing just that! Will make the change

Reply:
Done.